libraries/AP_Arming/AP_Arming.h

```diff
@@ -96,6 +96,7 @@ class AP_Arming {
     // these functions should not be used by Copter which holds the armed state in the motors library
     Required arming_required() const;
     virtual bool arm(AP_Arming::Method method, bool do_arming_checks=true);
+    virtual bool arm_force(AP_Arming::Method method) { return arm(method, false); }
```

Comment:
It would be preferable to check the number of arguments being passed to the C function (`lua_gettop`?) rather than creating methods like this, IMO.